pystats_docs.md

```diff
@@ -133,4 +133,4 @@ The number of times an unsupported opcode caused a potential trace to be truncat
 - set eval frame func: Setting the PEP 523 frame eval function,
   `_PyInterpreterState_SetFrameEvalFunc()`
 - builtin dict: Modifying the builtins, `__builtins__.__dict__[var] = ...`
-- func modification: Modifying a function, e.g. `func.__defaults__ = ...`, etc.
\ No newline at end of file
+- func modification: Modifying a function, e.g. `func.__defaults__ = ...`, etc.
```

Comment:
I can’t tell what changed here?

Reply:
Oh weird.  I made this with the Github online editor, which I almost never do.  Must be doing something to the whitespace, I guess.